config/main.py

```diff
@@ -2007,6 +2007,24 @@ def asymmetric(ctx, interface_name, status):
 
     run_command("pfc config asymmetric {0} {1}".format(status, interface_name))
 
+#
+# 'pfc config priority' command
+#
+
+@pfc.command()
+@click.argument('interface_name', metavar='<interface_name>', required=True)
+@click.argument('priority', type=click.Choice(['3', '4']))
```

Comment:
Looking forward, is there a way not to hard-code the lossless TCs?

Reply:
@wendani How about using 0-7? I think almost all ASICs only support 8 priorities.

Reply:
@wendani Support PFC on 0-7 now.